scipy/spatial/transform/rotation.py

```diff
@@ -4,14 +4,16 @@
 import scipy.linalg
 import warnings
 
+
 class Rotation(object):
     """Rotation in 3 dimensions.
 
-    This class will include initializers from different representations, converters
-    and some useful algorithms such as quaternion slerp and rotation estimation.
+    This class will include initializers from different representations,
+    converters and some useful algorithms such as quaternion slerp and
+    rotation estimation.
 
-    For initializing Rotations usage of `from_...` methods such as `from_quaternion`
-    is recommended instead of using `__init__`.
+    For initializing Rotations usage of `from_...` methods such as
```

Comment:
I think it is not very precisely put, basically `__init__` initializes from quaternions only.

I would write simply something like: ``"To create an instance use one of `Rotation.from_...` class(or factory?) methods."`` My point is that "recommended" is not a right word here. I think my short phrase explains everything is there to know.